modules/core/shared/src/main/scala/gem/math/MagnitudeValue.scala

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.math
+
+import cats.{ Order, Show }
+import cats.instances.int._
+
+/**
+ * Exact magnitude value represented as an int with the original value scaled up
+ * @param value This magnitude integral value
+ */
+final case class MagnitudeValue(private[gem] val scaledValue: Int) extends Product with Serializable {
```

Comment:
I don't think `scaledValue` needs to be private (and there's no point really because you can still access it with pattern-matching, although `sealed abstract case class` would remedy this).

To be consistent with similar classes I'd prefer this member be called `toCentiwhatever` (whatever the units are, which should also be noted in the class doc) and similarly `toDouble` should be `toDoubleWhatever`.

Reply:
Magnitude doesn't really have units

Reply:
Huh, ok. It's weird but I guess it's fine as-is. Can we make a note in the doc that it's dimensionless and maybe link to an article somewhere? It will be helpful to future me.